src/Explorer.js

```diff
@@ -1269,42 +1516,111 @@ class Explorer extends React.PureComponent<Props, State> {
             'Consolas, Inconsolata, "Droid Sans Mono", Monaco, monospace',
         }}
         className="graphiql-explorer-root">
-        {queryFields ? (
-          <RootView
-            fields={queryFields}
-            operation="query"
-            parsedQuery={parsedQuery}
-            onEdit={this._onEdit}
-            schema={schema}
-            getDefaultFieldNames={getDefaultFieldNames}
-            getDefaultScalarArgValue={getDefaultScalarArgValue}
-            makeDefaultArg={makeDefaultArg}
-          />
-        ) : null}
-        {mutationFields ? (
-          <RootView
-            fields={mutationFields}
-            operation="mutation"
-            parsedQuery={parsedQuery}
-            onEdit={this._onEdit}
-            schema={schema}
-            getDefaultFieldNames={getDefaultFieldNames}
-            getDefaultScalarArgValue={getDefaultScalarArgValue}
-            makeDefaultArg={makeDefaultArg}
-          />
-        ) : null}
-        {subscriptionFields ? (
-          <RootView
-            fields={subscriptionFields}
-            operation="subscription"
-            parsedQuery={parsedQuery}
-            onEdit={this._onEdit}
-            schema={schema}
-            getDefaultFieldNames={getDefaultFieldNames}
-            getDefaultScalarArgValue={getDefaultScalarArgValue}
-            makeDefaultArg={makeDefaultArg}
-          />
-        ) : null}
+        {relevantDefinitions.map((definition, index) => {
+          const operation:
```

Comment:
Why redefine definition to operation?

Reply:
Fixed. I was (and still am, somewhat) confused about `definition` vs `operation`